### ISSUE TITLE
Consider adding the proc macros to crate root

### ISSUE THREAD

**Issue by jimmycuadra** [alexcrichton/futures-await#8]
I'm probably in the minority on this, but I never use glob imports, because I like to see the source of all names, even if it's verbose. Currently, my code using futures usually looks something like this:

``` rust
extern crate futures;

use futures::{Future, IntoFuture, Stream};
```

When using futures-await, I now have to either change it to this:

``` rust
extern crate futures_await as futures;

use futures::prelude::{async, await, Future, IntoFuture, Stream};
```

or this:

``` rust
extern crate futures_await as futures;

use futures::{Future, IntoFuture, Stream};
use futures::prelude::{async, await};
```

It'd be nice if the `async` macro was available from the crate root instead of just the prelude module so I could just do this:

``` rust
extern crate futures_await as futures;

use futures::{async, await, Future, IntoFuture, Stream};
```

tl;dr, adding `async`, `await`, and `await_block` to the crate root makes it easier to drop in futures-await for futures without having to rewrite imports.

**Comment by alexcrichton** [comment]
Heh so my *ideal* solution here would actually be to add a `prelude` module to the `futures` crate which does the same thing as this crate...

**Comment by jimmycuadra** [comment]
That'd work too. Any reason not to do that?

**Comment by alexcrichton** [comment]
[Nope](https://github.com/alexcrichton/futures-rs/pull/568)!

**Comment by alexcrichton** [comment]
Prelude is now added!
